app/models/teacher_training_adviser/steps/overseas_timezone.rb

```diff
@@ -0,0 +1,24 @@
+module TeacherTrainingAdviser::Steps
+  class OverseasTimezone < Wizard::Step
+    attribute :time_zone, :string
+    attribute :telephone, :string
+
+    validates :time_zone, presence: { message: "Select a time zone" }
+    validates :telephone, telephone: true, presence: { message: "Enter a telephone number" }
+
+    def reviewable_answers
+      {
+        "time_zone" => time_zone.to_s,
```

Comment:
These are already strings, so shouldn't need to `to_s` them

Reply:
👍